scss/_patterns_form-password-toggle.scss

```diff
@@ -0,0 +1,34 @@
+@import 'settings';
+@import 'base_placeholders';
+@import 'patterns_icons';
+@include vf-b-placeholders;
+@include vf-p-icon-hide;
+@include vf-p-icon-show;
```

Comment:
@steverydz  Are these imports/includes needed here?

We don't import or include anything in patterns CSS, because it will be already provided by Vanilla in base styles.
I guess these imports/includes may be the reason the file size is so much larger in this PR.

Reply:
@bartaz I can remove placeholders as they were only needed when I was using the button link styles. The icon patterns are needed though, otherwise we don't have the mixins for the show and hide icons.